src/components/editor/property/InputListLOC.jsx

```diff
@@ -138,7 +141,7 @@ InputListLOC.propTypes = {
       useValuesFrom: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
     }),
   }).isRequired,
-  displayValidations: PropTypes.bool,
+  reduxPath: PropTypes.array,
```

Comment:
at first i made `reduxPath` a `PropTypes.array.required`, but tests failed because `InputListLOC.test.js` wasn't passing it in.  but it seems like IRL, the parent component always passes it in, and it's required in the parent component, so maybe `.required` would be more appropriate, and the tests should then be changed to reflect that?

if that's the case, i'm inclined  to ticket that as follow on work.  this still seems like an improvement, since there's now at least type checking where there previously wasn't any.

Reply:
While working on #694, I bumped into similar stuff.  Did you mean `.isRequired` as is on the propertyTemplate on line 143?  

Also https://github.com/LD4P/sinopia_editor/commit/5a0090af5656b986fb6c29375f23dfbf067ce0f1#diff-93b04186261f1a6fa9e8f243d588ef94 removed `.isRequired` from propertyTemplate and I found I had to do that while working on #694

Reply:
tried `isRequired` too, but same result.  might investigate as follow-on work.

Reply:
filed #901